test/services/PspDashboardServiceSpec.scala

```diff
@@ -76,12 +76,24 @@ object PspDashboardServiceSpec extends SpecBase with MockitoSugar {
       )
     )
 
-  private def schemeCard: CardViewModel =
+  private def schemeCard: CardViewModel = {
+    val baseLinks = Seq(
```

Comment:
Please use the output values for unit tests, not Messages calls and application router